spec/complete_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
 
-describe Conjur::CLI::Complete do
+describe Conjur::CLI::Complete, wip: true do
```

Comment:
Dunno whether they're being used, but it didn't feel right to rip them out completely (like I did for `env` support). 

I think `wip` is good for now, we can revisit in a separate story.